src/components/__tests__/AlertBar.test.tsx

```diff
@@ -0,0 +1,18 @@
+/// <reference types="cypress"/>
+
+import React from "react";
+import ReactDom from "react-dom";
+import { mount } from "@cypress/react";
+import { snackbarMachine } from "../../machines/snackbarMachine";
+import { useMachine } from "@xstate/react";
+
+import AlertBar from "../AlertBar";
+
+describe("smoke", () => {
+  it("works", () => {
+    const [, , snackbarService] = useMachine(snackbarMachine);
```

Comment:
BTW @JessicaSachs that is not the ReactDom issue. You can not just call the react hook inside the test